public/index.html

```diff
@@ -132,10 +132,12 @@ <h4 class="ui horizontal container divider header">
             </div>
 
             <h4 class="ui horizontal container divider header">
-                <i class="info circle icon"></i>
-                &nbsp;OPass 2016 - {{nowYear}} |
-                <a href="privacy-policy.html">Privacy Policy</a> |
-                <a href="mailto:dev@opass.app">Contact Us</a>
+                <div class="footer">
+                    <i class="info circle icon"></i>
+                    &nbsp;OPass 2016 - {{nowYear}} |
+                    <a href="privacy-policy.html">Privacy Policy</a> |
+                    <a href="mailto:dev@opass.app">Contact Us</a>
+                </div>
```

Comment:
so here still edited for what?

Reply:
I've added `<div class="footer">` for footer with margin.

Reply:
do you know you can just add that style into h4 element by a new class name ?